FinniversKit/Sources/Components/NMPInfobox/NMPInfoboxView.swift

```diff
@@ -0,0 +1,201 @@
+
+        public init(
+            informationType: NMPInfoboxView.InformationType,
+            title: String,
+            detail: String,
+            linkInteraction: Interaction? = nil,
+            primaryButtonInteraction: Interaction? = nil,
+            secondaryButtonInteraction: Interaction? = nil
+        ) {
+            self.informationType = informationType
+            self.title = title
+            self.detail = detail
+            self.linkInteraction = linkInteraction
+            self.primaryButtonInteraction = primaryButtonInteraction
+            self.secondaryButtonInteraction = secondaryButtonInteraction
+        }
+
+        var titleStyle: Label.Style {
+            .bodyStrong
+        }
+
+        var detailStyle: Label.Style {
+            .body
+        }
+    }
+}
+
+#warning("Og så skal der laves en SwiftUI version som rent faktisk bruger den der Infobox ting")
+public struct NMPInfoboxView: View {
+    public let viewModel: ViewModel
+
+    public init(viewModel: ViewModel) {
+        self.viewModel = viewModel
+    }
+
+    public var body: some View {
+        HStack(alignment: .top, spacing: 8) {
+
+            viewModel.informationType.icon
+
+            VStack(alignment: .leading, spacing: 8) {
+                Text(viewModel.title)
+                    .finnFont(viewModel.titleStyle)
+                Text(viewModel.detail)
+                    .finnFont(viewModel.detailStyle)
+
+                if let linkInteraction = viewModel.linkInteraction {
+                    SwiftUI.Button(
+                        action: { linkInteraction.onTapped() },
+                        label: {
+                            Text(linkInteraction.title).underline()
+                        }
+                    )
+                    .buttonStyle(InlineFlatStyle())
+                }
+
+                HStack(spacing: 8) {
+                    if let primaryButtonInteraction = viewModel.primaryButtonInteraction {
+                        SwiftUI.Button(
+                            action: { primaryButtonInteraction.onTapped() },
+                            label: {
+                                Text(primaryButtonInteraction.title)
+
+                            }
+                        )
+                        .buttonStyle(DefaultStyle(fullWidth: false))
+                    }
+
+                    if let secondaryButtonInteraction = viewModel.secondaryButtonInteraction {
+                        SwiftUI.Button(
+                            action: { secondaryButtonInteraction.onTapped() },
+                            label: { Text(secondaryButtonInteraction.title) }
+                        )
+                        .buttonStyle(FlatStyle(fullWidth: false))
+                    }
+
+                    Spacer()
+                }
+            }
+
+        }
+        .padding(.spacingM)
+        .overlay( // border
+            RoundedRectangle(cornerRadius: 4)
```

Comment:
Not sure if the design system has predefined radius constants? 🤔

Reply:
It doesn't look so. I can use `.spacingXS`, that matches 4

Reply:
Not sure that's a good idea. If for some reason the spacing gets changed the corner radius will change also, which is not correct. We need a new set of constants for radius if we need it. For now just leave it to what the design says it should be. You can ask the design team if the design system has or should have corner radius definitions.

Reply:
Fair point, I can live with that 🤝